cobra/flux_analysis/loopless.py

```diff
@@ -272,18 +272,15 @@ def construct_loopless_model(cobra_model):
             thermo_stoic["thermo_var_" + met.id][bound_id] = stoic
         # I * 1000 > v --> I * 1000 - v > 0
         reaction_ind = Reaction(reaction.id + "_indicator")
-        reaction_ind.variable_kind = "integer"
```

Comment:
Please remove the entire function `construct_loopless_model`and its reference in `cobra/flux_analysis/__init__.py`. Thanks!